src/Test/Diagnostics/project.lock.json

```diff
@@ -113,255 +114,254 @@
         "lib/portable-net45+win8+wp8+wpa81/System.Composition.Runtime.XML",
         "lib/portable-net45+win8+wp8+wpa81/System.Composition.TypedParts.dll",
         "lib/portable-net45+win8+wp8+wpa81/System.Composition.TypedParts.XML",
-        "License-Stable.rtf",
-        "Microsoft.Composition.nuspec",
-        "package/services/metadata/core-properties/12e2c94035674b25b911285cc26e3260.psmdcp"
+        "package/services/metadata/core-properties/12e2c94035674b25b911285cc26e3260.psmdcp",
+        "[Content_Types].xml"
       ]
```

Comment:
What scares me is that you just changed the contents of the files.  That shouldn't have changed project dependencies.